XenModel/Messages.ja.resx

```diff
@@ -14613,4 +14613,7 @@ VM のパフォーマンスを最適化するために、VCPU の数を物理 CP
   <data name="YOU_ARE_HERE" xml:space="preserve">
     <value>現在の位置</value>
   </data>
+  <data name="HEALTHCHECK_CANNOT_ALTER_SETTINGS" xml:space="preserve">

```

Comment:
Changing the l10n resx is not necessary at this stage, the translations are handled afterwards.